.chloggen/add_inspect_metrics.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: dockerstatsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Add cpu limit, cpu shares and restart count metrics from docker inspect api"
```

Comment:
Would it make sense to actually spell out the metrics being added? It may be more useful to users upgrading their Collector. That way, they can immediately start using them rather than starting to dig around to find out their names.

Reply:
Good point @gbbr. Fixed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803/commits/250dd380a647fc29b5722fd2df2b92e7ae20d2da